.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
```

Comment:
Why?

Reply:
It seems that we use the ```sudo``` related command in the ```before_script```.
We need to use the ```true```.